Library/Formula/homebank.rb

```diff
@@ -26,7 +26,7 @@ def install
     args << "--with-ofx" if build.with? 'libofx'
 
     system "./configure", *args
-    system "chmod +x ./install-sh"
+    chmod("+x", "install-sh")
```

Comment:
This *is* what is causing the failure:

```
Error: can't convert String into Integer
Please report this bug:
    http://git.io/brew-troubleshooting
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:1223:in `chmod'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:1223:in `chmod'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:876:in `chmod'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:875:in `each'
/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/fileutils.rb:875:in `chmod'
/usr/local/Library/Formula/homebank.rb:29:in `install'
/usr/local/Library/Homebrew/build.rb:136:in `install'
/usr/local/Library/Homebrew/formula.rb:410:in `brew'
/usr/local/Library/Homebrew/formula.rb:768:in `stage'
/usr/local/Library/Homebrew/resource.rb:91:in `unpack'
/usr/local/Library/Homebrew/extend/fileutils.rb:21:in `mktemp'
/usr/local/Library/Homebrew/resource.rb:88:in `unpack'
/usr/local/Library/Homebrew/resource.rb:81:in `stage'
/usr/local/Library/Homebrew/formula.rb:766:in `stage'
/usr/local/Library/Homebrew/formula.rb:406:in `brew'
/usr/local/Library/Homebrew/build.rb:114:in `install'
/usr/local/Library/Homebrew/build.rb:185
```

FileUtils.chmod in Ruby 1.8 does not accept string representation of the mode.